### ISSUE TITLE
ansi-html dependency vulneratiblity

### ISSUE THREAD

**Issue by mkusold** [storybookjs/storybook#16637]
**Describe the bug**
@storybook/react `6.3.12` has a dependency on `@pmmmwh/react-refresh-webpack-plugin` which has a dependency on `ansi-html`. There is currently a [security issue with this package](https://github.com/advisories/GHSA-whgm-jr23-g3j9).
"This affects all versions of package ansi-html. If an attacker provides a malicious string, it will get stuck processing the input for an extremely long time."
`react-refresh-webpack-plugin` [switched to using a similar community supported package](https://github.com/pmmmwh/react-refresh-webpack-plugin/pull/501) in order to fix this issue, but now we just need to upgrade @storybook/react to use the new version in [react-refresh-webpack-plugin 0.5.0](https://github.com/pmmmwh/react-refresh-webpack-plugin/releases/tag/v0.5.0)

**To Reproduce**
`yarn audit` or `npm audit` or check dependabot

**System**
```
Environment Info:

  System:
    OS: macOS 11.5.2
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
  Binaries:
    Node: 14.17.0 - ~/.nvm/versions/node/v14.17.0/bin/node
    Yarn: 1.22.10 - /usr/local/bin/yarn
    npm: 7.16.0 - ~/.nvm/versions/node/v14.17.0/bin/npm
  Browsers:
    Chrome: 95.0.4638.69
    Edge: 95.0.1020.44
    Firefox: 92.0.1
    Safari: 14.1.2
  npmPackages:
    @storybook/addon-a11y: ^6.3.7 => 6.3.7 
    @storybook/addon-actions: ^6.3.7 => 6.3.7 
    @storybook/addon-essentials: ^6.3.7 => 6.3.12 
    @storybook/addon-links: ^6.3.7 => 6.3.7 
    @storybook/addon-notes: ^5.3.21 => 5.3.21 
    @storybook/react: ^6.3.12 => 6.3.12 
```

**Comment by mkusold** [comment]
PS: the short term workaround is adding a resolutions onto your package.json
```
...
  "resolutions": {
    "@storybook/react/@pmmmwh/react-refresh-webpack-plugin": "0.5.0"
  },
...
```

**Comment by arumuganainar73** [comment]
We Developed Nx monorepo application and added storybook.

While running yarn audit command below packages are creating high vulanarbilty.

Command
I tried 'yarn audit --level high'

Failure Logs
@storybook/builder-webpack5
@angular-devkit/build-angular
@storybook/angular
@storybook/addon-essentials

**Comment by shilman** [comment]
@mkusold any chance you can submit a PR that upgrades the dependency?

**Comment by mkusold** [comment]
Yeah let me TAL and get setup with the contribution process. Ideally it should be pretty straightforward

**Comment by mkusold** [comment]
I got all setup to do the fix, and then saw it was already done in v 6.4.0-beta.31
as a result of https://github.com/storybookjs/storybook/pull/16155. So I'm going to close out this issue.
